File: brenda028/programação_basica/questao08.py
print ("** Programa de volume das formas geometricas ** ")
l = float(input("Digite um lado do cubo:"))
v = l * l * l 
print("O volume do cubo sera" + str (v))

print(" ** Volume de um paralelepipedo ** ")
c = float(input("Diite o comprimento:"))
l = float(input("Digite a largura:"))
a = float(input("Digite a altura:"))
y = c * l * a
print("O volume do paralelepipedo" + str (y))

print("** Volume de uma esfera **")
r = float(input("Informe o raio da esfera:"))
w = 4 * 3.14 * r * r * r / 3 
print("O volume da esfera eh: " + str (w))

File: juvenal030/programacao_basica/Questao08.java
import java.util.Scanner;

public class Questao08 {
    public static void main (String [] args) {
        Scanner s;
        double a;
        double b;
        double c;
        double d;
        double e;
        double f;
        double g;
        double h;

        
        s = new Scanner (System.in);
        System.out.println("** volume do cubo**");
        System.out.println(" digite o lado do cubo ");
        a = s.nextDouble ();
        b = a * a * a;
        System.out.println(" o volume do cubo sera " + b);

        System.out.println("** volume do paralelepipedo**");
        System.out.println(" digite a altura do paralelepipedo ");
        c = s.nextDouble ();
        System.out.println(" digite o comprimento do paralelepipedo");
        d = s.nextDouble ();
        System.out.println(" digite a base do paralelepipedo");
        e = s.nextDouble ();
        f = c * d * e;
        System.out.println(" o volume do paralelepipedo " + f );

        System.out.println("** volume da esfera **");
        System.out.println(" informe o raio da esfera : ");
        g = s.nextDouble ();
        h = 4 * 3.14 * g * g * g/3;
    }
}
